app/views/internal/configs/show.html.erb

```diff
@@ -1,238 +1,282 @@
-                           class: "form-control",
-                           value: SiteConfig.periodic_email_digest_max,
-                           placeholder: 0 %>
-        <div class="alert alert-info">Determines the maximum for the periodic email digest</div>
+      <div class="card mt-3">
+        <div class="card-header">Tags</div>
+        <div class="card-body">
+          <div class="form-group">
+            <%= f.label :suggested_tags %>
+            <%= f.text_field :suggested_tags,
+                             class: "form-control",
+                             value: SiteConfig.suggested_tags.join(","),
+                             placeholder: "List of valid tags: comma separated, letters only e.g. beginners,javascript,ruby,swift,kotlin" %>
+            <div class="alert alert-info">Determines which tags are suggested to new users during onboarding (comma
+              separated, letters only)</div>
+          </div>
+        </div>
       </div>
 
-      <div class="form-group">
-        <%= f.label :periodic_email_digest_min %>
-        <%= f.number_field :periodic_email_digest_min,
-                           class: "form-control",
-                           value: SiteConfig.periodic_email_digest_min,
-                           placeholder: 2 %>
-        <div class="alert alert-info">Determines the mininum for the periodic email digest</div>
+      <div class="form-group mt-3">
+        <%= f.submit "Update Site Configuration", class: "btn btn-primary" %>
       </div>
+      <% end %>
     </div>
-  </div>
 
-  <div class="card mt-3">
-    <div class="card-header">Tags</div>
-    <div class="card-body">
-      <div class="form-group">
-        <%= f.label :suggested_tags %>
-        <%= f.text_field :suggested_tags,
-                         class: "form-control",
-                         value: SiteConfig.suggested_tags.join(","),
-                         placeholder: "List of valid tags: comma separated, letters only e.g. beginners,javascript,ruby,swift,kotlin" %>
-        <div class="alert alert-info">Determines which tags are suggested to new users during onboarding (comma separated, letters only)</div>
+    <div class="card w-100">
+      <div class="card-header" id="appConfigHeader">
+        <h2 class="d-inline">Environment variables / AppConfig (readonly)</h2>
+        <button class="btn btn-secondary float-right" type="button" data-toggle="collapse"
+          data-target="#appConfigBodyContainer" aria-expanded="false" aria-controls="appConfigBodyContainer">
+          Toggle
+        </button>
+      </div>
+      <div id="appConfigBodyContainer" class="collapse hide" aria-labelledby="appConfigHeader">
+        <table class="table">
+          <thead>
+            <tr>
+              <th>Variable</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <% ENVied.config.variables.each do |v| %>
+            <% if v.group.in?(ENVied.required_groups) %>
```

Comment:
Do we need the check?

Reply:
If we want this to work in development, then yes. `ENVied.variables` will return them all, but the object won't respond to the method until the relevant group has been required. So locally the page crashes with a `NoMethodError`. The other way is checking for `ENVied.responds_to?(v.name)`, but I liked the group check better, as it somewhat documents why the object might not have a certain method available.